.travis.yml

```diff
@@ -18,6 +18,7 @@ script:
   - docker build -t ${REPO}:${TAG} .
   - docker run --rm -ti ${REPO}:${TAG} clang-8 --version
   - docker run --rm -ti ${REPO}:${TAG} qemu-system-x86_64 --version
+  - docker run --rm -ti ${REPO}:${TAG} qemu-system-ppc --version
```

Comment:
Sorry, are these commands printing the versions of our qemu binaries at Docker image creation time?

Reply:
Yes and making sure that they are actually available.